src/bsp/platform/stm32fx_nucleo_wifi/xi_bsp_tls_certs.h

```diff
@@ -0,0 +1,101 @@
+#define GLOBALSIGN_ROOT_CERT "-----BEGIN CERTIFICATE-----\n\
```

Comment:
Are these certificates part of the xively library, too?  We might be doubling up the storage space here.